app/assets/javascripts/controllers/ansible_repository/repository_form_controller.js

```diff
@@ -124,7 +124,7 @@ ManageIQ.angular.app.controller('repositoryFormController', ['repositoryId', 'mi
   };
 
   var getManagerResource = function(response) {
-    if (!response.resources.length) {
+    if (! response.resources.length) {
```

Comment:
inconsistent with the change: https://github.com/ManageIQ/manageiq-ui-classic/pull/4860/commits/aa983f9d8c9467daa969c9804943fc0095afad07#diff-9a3287187411aea00b9bc8b08324b5edR125

personally, I'd choose no space between negation and the expression.

EDIT: from the changes below, I guess the preferred option is to use the space.

Reply:
yes it is difference between ES5 and ES6 i will unify them (no space)